package.json

```diff
@@ -16,6 +16,9 @@
   "dependencies": {
     "@sphereon/pex": "^0.6.3-unstable.1",
```

Comment:
0.7.0 is out, which should not really impact the integration, but contains several fixes

Reply:
I will update it.